posthog/temporal/common/clickhouse.py

```diff
@@ -723,16 +725,22 @@ async def astream_query_in_batches(
             query: The SQL query to execute. Must end with FORMAT TabSeparatedWithNamesAndTypes.
             query_parameters: Optional query parameters to interpolate.
             query_id: Optional ClickHouse query ID.
-            batch_size: The number of rows per batch to yield.
+            batch_size: The number of rows per batch to yield. Either `batch_size` or `batch_size_mb` must be set.
+            batch_size_mb: The max size of the batch to yield. Either `batch_size` or `batch_size_mb` must be set.
```

Comment:
Maybe we should indicate which one wins if they are both set in the comment here?